gateway-service-definitions/src/main/resources/services/livy/0.4.0/rewrite.xml

```diff
@@ -45,4 +45,8 @@
     <rewrite template="{$frontend[url]}/yarn/nodemanager/node/containerlogs/{**}?{scheme}?host={$hostmap(host)}?{port}"/>
   </rule>
 
+  <rule dir="OUT" name="LIVYSERVER/livy/outbound/headers/ui" pattern="{scheme}://{host}:{port}/ui/">
```

Comment:
Hmm I guess I wasn't completely clear and was talking about only the `rewrite template` not the pattern here. Can you double check that looking for `/ui/` as the pattern is correct still? I want to make sure the header we are looking for matches in all the cases.

Reply:
Yes I verified with these changes that looking for `/ui/` as the pattern is correctly redirecting to livy ui page.